docs/tutorials/github.md

```diff
@@ -0,0 +1,93 @@
+# GitHub Tutorial
+
+This tutorial provides an introduction to using `trestlebot` with GitHub.  We will be using a single GitHub repository for our trestle authoring workspace and executing the `trestlebot` commands as GitHub actions.  Note, each repo is intended to support authoring a single OSCAL model type (SSP, component definition, etc.).  If authoring more than one, then a dedeicated repository should be used for each model.
+
+
+### 1. Prerequisites
+
+Before moving on, please ensure you have completed the following:
+
+1. create a new (or use an existing) empty GitHub repository
+2. clone the repo to your local workstation
+3. clone the [trestle-bot](https://github.com/RedHatProductSecurity/trestle-bot/tree/main) repo to your local workstation and run `poetry install`
+
+
+### 2. Set Permissions for GitHub Actions
+
+The `trestlebot` commands will be run inside of GitHub actions.  These commands often perform `write` level operations against the repo contents.  The following repo settings need to be in place to support these actions.
+
+1. Click the `Settings` tab for your GitHub repo 
+2. Select `Actions` -> `General` from the left-hand menu
+3. Scroll down to `Workflow permissions`
+4. Ensure `Read repository contents and packages permissions` is selected
+5. Ensure `Allow GitHub Actions to create and approve pull requests` is checked
+
+### 3. Initialize trestlebot Workspace
+
+We will now use the `trestlebot init` command to initialize our emtpy GitHub repository.  Unlike the other trestlebot commands, this command is run on your local workstation.  The trestlebot commands can be installed by cloning the [trestle-bot](https://github.com/RedHatProductSecurity/trestle-bot/tree/main) repo and running `poetry install`.  For this tutorial we will be authoring a component-definition.
```

Comment:
Perhaps we also add a podman command to show how to use init and it is the default entrypoint on the `trestlebot` image? What do you think?

Reply:
Currently there is not a container entrypoint defined for `init` under `actions/` but I can add that here if you'd like?

Reply:
I actually meant that if you run `trestle-bot` from the built container image, the default [entrypoint](https://github.com/RedHatProductSecurity/trestle-bot/blob/d1490cbde72b204875260cd210f61760e9f3c056/Dockerfile#L63) in the image would run the init command.